shell/platform/common/cpp/text_input_model.cc

```diff
@@ -78,9 +75,9 @@ void TextInputModel::AddCodePoint(char32_t c) {
 
 void TextInputModel::AddText(const std::u16string& text) {
   DeleteSelected();
-  text_.insert(selection_extent_, text);
-  selection_extent_ += text.length();
-  selection_base_ = selection_extent_;
+  auto position = selection_.position();
```

Comment:
Minor nit for all the uses of `auto` in this change; it's non-obvious from just reading the RHS what the exact type of this is (e.g., whether or not it is signed), so I think using the actual type rather than `auto` would be more consistent with style guide guidance.

Reply:
Done.